esmvalcore/experimental/config/_config_object.py

```diff
@@ -0,0 +1,191 @@
+"""Importable config object."""
+
+import os
+from datetime import datetime
+from pathlib import Path
+
+import yaml
+
+import esmvalcore
+
+from ._config_validators import _validators
+from ._validated_config import ValidatedConfig
+
+
+class Config(ValidatedConfig):
+    """ESMValTool configuration object."""
+
+    validate = _validators
```

Comment:
This could probably be made private, I think this will only confuse users
```suggestion
    _validate = _validators
```

Reply:
Yeah, good point. I was wondering why `validate` showed up on the docs all of a sudden...